src/chargen.cpp

```diff
@@ -825,6 +844,11 @@ int parse_prestige_race(struct descriptor_data *d, const char *arg)
   case 'd':
   case 'D':
     return RACE_FEATHERED_SERPENT;
+  case 'o':
+  case 'O':
+    d->ccr.is_otaku = !d->ccr.is_otaku;

```

Comment:
Are any races forbidden from being otaku (e.g. dragons, because they cannot have cyberware implanted)? If so, you'll want to check here for that race already being set, and check in the race toggle and refuse to proceed if otaku is already enabled.

Reply:
From discord: we're going to lock out all but the common core metatypes